Sources/Core/Client/ClientError.swift

```diff
@@ -74,25 +74,56 @@ public enum ClientError: LocalizedError {
             
         case .requestFailed(let error):
             if let error = error {
-                return "A request failed: \(error)"
+                return "A request failed: \(error.localizedDescription)"
             }
             
             return "A request failed with unknown error"
             
         case .responseError(let error):
-            return "A response failed: \(error)"
-        case .encodingFailure(let error, let object):
-            return "A encoding failed: \(error) for object: \(object)"
+            return error.localizedDescription
+        case .encodingFailure(let error, _):
+            return "An encoding failed: \(error.localizedDescription)"
         case .decodingFailure(let error):
-            return "A decoding failed: \(error)"
+            return "A decoding failed: \(error.localizedDescription)"
         case .errorMessage(let message):
             return message.text
         }
     }
+
+    public var debugDescription: String {
```

Comment:
Since this is `debugDescription`, can we make sure all the info are there? For example, `unexpectedError` has `Error?` wrapped but it's not used here. It could be:
```swift
case .unexpectedError(let description, let error):
  return "ClientError.unexpectedError(\(description):\(error))"
```
Same for all other cases missing wrapped values.

Reply:
`Error?` was not part of `unexpectedError` but I added it :)